src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/BookingCalculationService.kt

```diff
@@ -90,9 +90,9 @@ class BookingCalculationService(
      However here we reduce the list by any duplicated offences with the same release conditions, to improve calculation time.
    */
   private fun collapseDuplicateConsecutiveSentences(consecutiveSentences: List<ConsecutiveSentence>): List<ConsecutiveSentence> {
-    return consecutiveSentences.distinctBy {
-      it.orderedSentences.joinToString { sentence -> "${sentence.sentencedAt}${sentence.identificationTrack}${sentence.totalDuration()}${sentence.javaClass}" }
-    }
+    return consecutiveSentences
+      .sortedBy { it.getOrderedIdentifiers(true) }
```

Comment:
super minor, but it may be easier to have context if you specify **includeCalculation = true** without navigating to the method.